llama_cpp/llama.py

```diff
@@ -445,7 +444,7 @@ def sample(
         mirostat_eta: float = 0.1,
         mirostat_tau: float = 5.0,
         penalize_nl: bool = True,
-        logits_processors=None
+        logits_processors:  List[Callable[[List[llama_cpp.c_int], List[llama_cpp.c_float]], List[float]]] = None
```

Comment:
Name should be changed to `logits_processor` to match hugginface generate argument in https://huggingface.co/docs/transformers/v4.26.0/en/main_classes/text_generation#transformers.GenerationMixin.generate.logits_processor

Additionally, the types here should be changed to python `float` and `int`